src/components/InvertedFlatList/index.js

```diff
@@ -63,7 +44,6 @@ class InvertedFlatList extends React.Component {
                 ref={el => this.list = el}
                 shouldMeasureItems
                 contentContainerStyle={StyleSheet.compose(this.props.contentContainerStyle, styles.justifyContentEnd)}
-                style={{transform: [{translateX: 0}, {translateY: 0}, {scaleY: -1}]}}
```

Comment:
@dbond09 Can you show some clarity on why we don't need this also?

Reply:
When the `inverted` attribute is set on a FlatList, it is also passed to the underlying VirtualizedList, which already defines the scaleY transform that we need here.

https://github.com/necolas/react-native-web/blob/fa47f80d34ee6cde2536b2a2241e326f84b633c4/packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js#L2158-L2160

Reply:
Cool thanks!